package SanProto;

//开始聊天
message StartChatReq {
  enum TYPE {
    WORLD = 1;
    UNION = 2;
  }

  required TYPE type = 1;
  optional int32 union_id = 2;
}
message StartChatRes {
  required int32 status = 1;
  optional bool available = 2;
  optional bytes hostname = 3;
  optional int32 port = 4;
  optional bytes roomname = 5;
  optional bytes password = 6;
}

//管理聊天（封禁、解禁）
message ManageChatReq {
  required uint64 target_user_id = 1;
  required bool enable = 2;
  optional int32 lock_min = 3;
}
message ManageChatRes {
  required int32 status = 1;
}

